src/index.html

```diff
@@ -1,13 +1,431 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="shortcut icon" href="./images/favicon.png">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="#menu"
+              class="icon icon--menu"
+            >
+            </a>
+            <a href="#" class="logo">
+              <img
+                src="./images/logo.png"
+                alt="Miami Condo Kings"
+                class="logo__image"
+                >
+            </a>
+            <a
+              href="tel:+1 234 555-55-55"
+              class="icon icon--phone header__call"
+            >
+            </a>
+          </div>
+          <h1 class="header__title">
+            Miami<br>
+            Condo Kings
+          </h1>
```

Comment:
Margin to bottom here is a bit wrong, https://prnt.sc/1tt39l5. It should be only 80px, you might wanna reset default margins 😉